Marlin/src/lcd/language/language_it.h

```diff
@@ -259,8 +259,8 @@ namespace LanguageNarrow_it {
   LSTR MSG_MESH_SAVED                     = _UxGT("Mesh %i salvata");
   LSTR MSG_MESH_ACTIVE                    = _UxGT("Mesh %i attiva");
   LSTR MSG_UBL_NO_STORAGE                 = _UxGT("Nessuna memoria");
-  LSTR MSG_UBL_SAVE_ERROR                 = _UxGT("Err: Salvataggio UBL");
-  LSTR MSG_UBL_RESTORE_ERROR              = _UxGT("Err: Ripristino UBL");
+  LSTR MSG_UBL_SAVE_ERROR                 = _UxGT("Salvataggio UBL Error");
```

Comment:
Does problem arise with 'error' or everything that starts with such text?
A better Italian translation would be "Errore salvataggio ULB"

Reply:
Checking OctoPrint sources, only messages beginning with "[Ee]rror:" trigger the error response, so most strings can be left alone.